staging/prometheus-operator/dependent-charts/kube-state-metrics/values.yaml

```diff
@@ -1,8 +1,8 @@
 # Default values for kube-state-metrics.
 prometheusScrape: true
 image:
-  repository: k8s.gcr.io/kube-state-metrics
-  tag: v1.5.0
+  repository: quay.io/coreos/kube-state-metrics
```

Comment:
I take it this change reflects upstream changes?

Reply:
Yes all the changes that are in the PR is what was already being deployed as part of `master`.